package.json

```diff
@@ -10,6 +10,7 @@
     "@formatjs/intl": "^2.0.0",
     "@giphy/js-components": "^5.0.5",
     "@giphy/js-fetch-api": "^5.3.0",
+    "@koa/bodyparser": "^5.0.0",
```

Comment:
The `@koa/bodyparser` is removed in the previous commit but the same version of that package is added back here, without any bump in the `PROVISION_VERSION`. Why is that?